presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestMapColumnStatisticsBuilder.java

```diff
@@ -50,7 +50,7 @@ public Object[][] keySupplier()
     @Test
     public void testAddMapStatisticsNoValues()
     {
-        MapColumnStatisticsBuilder builder = new MapColumnStatisticsBuilder();
+        MapColumnStatisticsBuilder builder = new MapColumnStatisticsBuilder(true);
```

Comment:
Changed this test to follow up on my mistake in the initial implementation of MapColumnStatisticsBuilder